### DISCUSSION TITLE
How to align header text. center or right?

### DISCUSSION THREAD

**Discussion by monzer15** [awcodes/filament-table-repeater#77] [General]
How to align header text. center or right?

**Comment by awcodes** [comment]
Not sure what you are asking.

Select the heading then hit one of the alignment buttons.

**Comment by monzer15** [comment]
![align](https://github.com/awcodes/filament-table-repeater/assets/59767837/da3bf908-ca36-43ca-99c8-f97d6516e64e)

I mean the headers, in the above picture the default text-align is left, is there any method like:

 $repeater->headersAlign('center');`
or 
$repeater->extraHeaderAttributues(['class' => 'text-center']); 

or something similar to this?

**Comment by awcodes** [comment]
Ah, currently, there's no method for that.

Could be a good PR. :)

**Reply by monzer15** [reply]
Ok, thank u for your time
